rules/windows/persistence_user_account_creation.toml

```diff
@@ -1,19 +1,48 @@
 [metadata]
 creation_date = "2020/02/18"
 maturity = "production"
-updated_date = "2021/09/23"
+updated_date = "2022/03/23"
 
 [rule]
 author = ["Elastic"]
 description = """
-Identifies attempts to create new local users. This is sometimes done by attackers to increase access to a system or
-domain.
+Identifies attempts to create new users. This is sometimes done by attackers to increase access to a system or domain.
 """
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "User Account Creation"
+note = """## Triage and analysis
+
+### Investigating User Account Creation
+
+Attackers may create new accounts (both Local and Domain) to maintain access to victim systems.
+
+This rule will monitor for the usage of `net.exe` to create new accounts.
+
+#### Possible investigation steps:
+
+- Investigate process execution chain (parent process tree).
+- Identify the user account which performed the action and whether it should perform this kind of action.
+- Identify if the account was added to privileged groups or assigned special privileges after creation.
+- Investigate other alerts related to the user/host in the last 48 hours.
+
+### False Positive Analysis
+
+- Account creation is a common administrative task, so there is a high chance of the activity being legitimate. Before
+the further investigation, it should be verified that this activity is not benign.
+
+### Related Rules
+
+- Creation of a Hidden Local User Account - 2edc8076-291e-41e9-81e4-e3fcbc97ae5e
+
+### Response and Remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+- Isolate the involved host to prevent further post-compromise behavior.
+- Delete the created account.
```

Comment:
```suggestion
- Delete the created account if unexpected.
- Reset password for the user account leveraged to create the new account.
```

Reply:
I think that Response actions needs to be taken only if the event is not an FP, so I think we don't need to put the 'if unexpected'